udata/core/dataservices/api.py

```diff
@@ -49,6 +53,44 @@ def post(self):
         return dataservice, 201
 
 
+@ns.route("/recent.atom", endpoint="recent_dataservices_atom_feed")
```

Comment:
Do you want to add a test on the dataservices feed?

Reply:
Done in 4d73be89a